code/datums/components/twohanded.dm

```diff
@@ -117,7 +117,7 @@
  * * user The mob/living/carbon that is wielding the item
  */
 /datum/component/two_handed/proc/wield(mob/living/carbon/user)
-	if(wielded)
+	if(wielded || !istype(user))
```

Comment:
Why are non carbons being passed in?

Reply:
this run timed on the first iteration of the TK block, it passed in the turf.
This issue is now resolved by prevent TK from wielding two handed items.